prisma/db.ts

```
@@ -66,8 +70,13 @@ export async function handlePrismaOperation<T>(
 const connectionString = process.env.DATABASE_URL;
 
 const adapter = new PrismaNeon({ connectionString });
-const prisma = new PrismaClient({ adapter });
 
-if (process.env.NODE_ENV === "development") global.prisma = prisma;
+const globalForPrisma = globalThis as unknown as {
+  prisma: PrismaClient | undefined;
+};
+
+const prisma = globalForPrisma.prisma ?? new PrismaClient({ adapter });
```

Biome (2.1.2)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)